src/main/java/ca/bc/gov/chefs/etl/forms/pcd/chc/pcPatientServices/processor/PcdChcPCPSApiResponseProcessor.java

```diff
@@ -0,0 +1,148 @@
+package ca.bc.gov.chefs.etl.forms.pcd.chc.pcPatientServices.processor;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.camel.Exchange;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import ca.bc.gov.chefs.etl.constant.Constants;
+import ca.bc.gov.chefs.etl.constant.PCDConstants;
+import ca.bc.gov.chefs.etl.core.model.IModel;
+import ca.bc.gov.chefs.etl.core.model.SuccessResponse;
+import ca.bc.gov.chefs.etl.core.processor.BaseApiResponseProcessor;
+import ca.bc.gov.chefs.etl.forms.pcd.chc.pcPatientServices.json.Root;
+import ca.bc.gov.chefs.etl.forms.pcd.chc.pcPatientServices.model.PCPSChcSubmission;
+import ca.bc.gov.chefs.etl.forms.pcd.chc.pcPatientServices.model.PCPSChcSubmissionData;
+import ca.bc.gov.chefs.etl.util.CSVUtil;
+import ca.bc.gov.chefs.etl.util.FileUtil;
+import ca.bc.gov.chefs.etl.util.JsonUtil;
+
+public class PcdChcPCPSApiResponseProcessor extends BaseApiResponseProcessor {
+  @SuppressWarnings("unchecked")
+  @Override
+  public void process(Exchange exchange) throws Exception {
+    String payload = exchange.getIn().getBody(String.class);
+    payload = JsonUtil.roundDigitsNumber(payload);
```

Comment:
You may want to check if we want to round numbers. We aren't doing so for budget/financials anymore.

Reply:
Removed from Processors as it is no longer needed.